extensions/mypy_extensions.py

```diff
@@ -93,5 +95,35 @@ class Point2D(TypedDict):
     """
 
 
+def Arg(typ=Any, name=None):
+    """A normal positional argument"""
+    return typ
```

Comment:
Add comment about why it makes sense to just return `typ` here and elsewhere in this file.